test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -72,7 +72,7 @@ public void ItDoesNotImplicitlyRestoreAProjectWhenTestingWithTheNoRestoreOption(
 
             new DotnetTestCommand()
                 .WithWorkingDirectory(testProjectDirectory)
-                .ExecuteWithCapturedOutput($"{TestBase.ConsoleLoggerOutputNormal} --no-restore")
+                .ExecuteWithCapturedOutput($"{TestBase.ConsoleLoggerOutputNormal} --no-restore /p:IsTestProject=true")
```

Comment:
Why was this change needed?

Reply:
Behavior change due to https://github.com/Microsoft/vstest/pull/1745

`IsTestProject` property added by `Microsoft.NET.Test.sdk` package. As the test trying to execute tests without restoring we are not getting expected exit code/error message. 

I think we should add info message when skipping a project if there is no `IsTestProject` property then update this test to asset on same info message. If this behavior sounds good we will take this change separately(next insertion).

Reply:
@livarcocc I agree with you. I have created https://github.com/Microsoft/vstest/issues/1775 to fix this test in next insertion. Can we merge this PR?

Reply:
I would rather not, as this is a regression in behavior.

Reply:
@livarcocc We can't fail the `dotnet test` command if there is no `IsTestProject` prop. The behavior change is intended we should remove this test.